### ISSUE TITLE
Add size checking to `table` inputs > 2

### ISSUE THREAD

**Issue by bkDJ** [uiua-lang/uiua#468]
```uiua
⧻♭ ⊞+. ⇡1e4       # errors because 1e8 is too big
⧻♭ ⊞(+++)... ⇡1e2 # would also be be 1e8, but does not error
```

**Comment by kaikalii** [comment]
Unfortunately, I don't know a good way to do this accurately with the way table is implemented. The output size for that instance of table is not known ahead of time.

**Comment by bkDJ** [comment]
Ah. I haven't looked at the implementation and was hoping it could just be changed to something like "join all argument shapes, reduce multiply, assert less than limit".

**Comment by bkDJ** [comment]
Mostly fixed as of e3756b5c5c35c1b0551dfc660c4e3c73b1052e74 ❤️
